nodepool/files/nodepool.yml

```diff
@@ -119,6 +123,8 @@ providers:
     cloud: public_cloud
     boot-timeout: 120
     hostname-format: nodepool-{label.name}-{provider.name}-{node.id}
+    # Ignore the quota given by public cloud and only respect what we provide as max-servers
+    ignore-provider-quota: True
```

Comment:
We're not using OnMetal in ORD, so perhaps we should just leave the ``ignore-provider-quota`` out of its pool configuration.